src/Stack/Types/Config.hs

```diff
@@ -500,6 +504,7 @@ instance FromJSON ConfigMonoid where
          configMonoidExtraIncludeDirs <- obj .:? "extra-include-dirs" .!= Set.empty
          configMonoidExtraLibDirs <- obj .:? "extra-lib-dirs" .!= Set.empty
          configMonoidConcurrentTests <- obj .:? "concurrent-tests"
+         configMonoidLocalBin <- obj .:? "local-bin"
```

Comment:
Should this be `"local-bin-path"` for consistency with the command-line argument name?